docs/_plugins/build_api_docs.rb

```diff
@@ -0,0 +1,205 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
So is the change basically refactoring?

Reply:
Yes. The goal is to preserve current behavior while making the code easier to maintain.